cinema_service/urls.py

```diff
@@ -5,4 +5,5 @@
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("__debug__/", include("debug_toolbar.urls")),
+    path("api/user/", include("user.urls", namespace="user"))
```

Comment:
Ensure that the `app_name` variable is defined in the `cinema/urls.py` and `user/urls.py` files to support namespacing, as this is crucial for URL routing.